<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bugRecordsClass</class>
 <widget class="QMainWindow" name="bugRecordsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bugRecords</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="figure_show">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>641</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="solution">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>931</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>解决方法</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_solution">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>21</y>
       <width>891</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="function">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>功能</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="next_question">
          <property name="text">
           <string>下一个</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="to_bottom">
          <property name="text">
           <string>直至底部</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addQuestion">
          <property name="text">
           <string>添加问题</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="last_question">
          <property name="text">
           <string>上一个</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_to_top">
          <property name="text">
           <string>回到顶部</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_solution">
          <property name="text">
           <string>添加解决方法</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="question_discription">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>问题描述</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_question">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>703</x>
      <y>441</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuv0_1">
    <property name="title">
     <string>v0.1</string>
    </property>
   </widget>
   <addaction name="menuv0_1"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bugRecords.qrc"/>
 </resources>
 <connections/>
</ui>
